### ISSUE TITLE
Dotnet file does not install

### ISSUE THREAD

**Issue by TobiasNeuhauser** [Gcenx/WineskinServer#64]
Hi, I am using Wineskin 2.9.0.7 RC1 with 64-bit engine 19.0.1, 5.8 or 5.19 on Mohave. I tried to install dotnet480 and dotnet452 via winetrick or via exe file but both ways do not work. Installing with engine 19.0.1 gives no error message, but installing with wine 5.x gives error message about installation not working. After that when I start ModOrganizer as testrun in the advanced section, it says dotnet file missing and therefore I cannot not start Oblivion/Skyrim/Fallout3 via ModOrganizer. Any ideas how to fix this?
Installing dotnet480 file on another laptop under Linux with PlayOnLinux via exe file works fine.

**Comment by Gcenx** [comment]
What this using a newly created wrapper?

Is there a valid reason you’d using a 64Bit engine over a 32Bit engine?

I’d avoid using dotnet452 instead use dotnet46.

I won’t get time to test this until I’m home but the first time you launch the winetricks menu it downloads the current version from GitHub, so comparability will vary depending of older workaround were removed.

Please note WineCX19 is a highly patched wine-4.12.1.

**Comment by TobiasNeuhauser** [comment]
I tried again, this time as you said with dotnet46 via winetricks. And yes, Oblivion was installed in a newly created wrapper. Although winetrick ran through fine successfully, right at installing dotnet46 after all the other required dotnet files files, there was an error message "could not find winemenubuilder.exe". Hence the dotnet file missing message in ModOrganizer persists and if ModOrganizer is run with Oblivion Scrip Extender (OBSE), it still crashes.  
Why using 64-bit? Because with 64-bit there is more RAM available and for modded Bugthesda Games it is really needed. Winecfg is set to Windows 7. Any ideas?
By the way, vanilla Oblivion runs fine in Wineskin, but who want to play vanilla Oblivion? Thanks

**Comment by Gcenx** [comment]
You can ignore `could not find winemenubuilder.exe` wine(64) uses that to create the .desktop files that are useless on macOS.

I have a feeling ModOrganizer needs .Net3.5 not the later .Net46

**Comment by TobiasNeuhauser** [comment]
This time I checked in winetrick dotnet35, dotnet40 and dotnet46. After installation, the error message in ModOrganizer about dotnet not installed or missing is gone, but it still crashed at the same time as before. It still crashes 20 seconds after starting Oblivion in ModOrganizer via Start OBSE. I tried with engine 19.0.1 and 5.19, does not work. Will try out again, when your new version is out, thanks again.
